tests/unit/coursier_utilities_test.bzl

```diff
@@ -12,18 +12,20 @@ escape_test = unittest.make(_escape_test_impl)
 
 def _get_classifier_test_impl(ctx):
     env = unittest.begin(ctx)
-    asserts.equals(env, None, get_classifier("groupId:artifactId:version"))
-    asserts.equals(env, None, get_classifier("groupId:artifactId:packaging:version"))
-    asserts.equals(env, "classifier", get_classifier("groupId:artifactId:packaging:classifier:version"))
+    asserts.equals(env, None, get_classifier("groupId:artifactId:1.2.3"))
+    asserts.equals(env, None, get_classifier("groupId:artifactId:packaging:1.2.3"))
+    asserts.equals(env, "classifier", get_classifier("groupId:artifactId:packaging:classifier:1.2.3"))
     return unittest.end(env)
 
 get_classifier_test = unittest.make(_get_classifier_test_impl)
 
 def _get_packaging_test_impl(ctx):
     env = unittest.begin(ctx)
-    asserts.equals(env, None, get_packaging("groupId:artifactId:version"))
-    asserts.equals(env, "packaging", get_packaging("groupId:artifactId:packaging:version"))
-    asserts.equals(env, "packaging", get_packaging("groupId:artifactId:packaging:classifier:version"))
+    asserts.equals(env, None, get_packaging("groupId:artifactId:1.2.3"))
+    asserts.equals(env, "packaging", get_packaging("groupId:artifactId:packaging:1.2.3"))
+    asserts.equals(env, "packaging", get_packaging("groupId:artifactId:packaging:classifier:1.2.3"))
+    asserts.equals(env, "packaging", get_packaging("groupId:artifactId:1.2.3@packaging"))
```

Comment:
Readability and testability suggestion: consider putting all the coordinates being tested in this file into a table, with their expected components, and test every coordinate for every component. This will improve our coverage for negative testing; like how line 27 should return None for classifier, but it's not being tested now.

Reply:
This is testing the `get_packaging` method from `coursier_utilties.bzl`, but I'll do that for the `coordinates_test.bzl` file, which feels like a more appropriate place.